src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/CodeReaderWriter/CodeModifierConfig.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.DotNet.MSIdentity.CodeReaderWriter
+{
+    public class CodeChange
+    {
+        public string? InsertAfter { get; set; }
+        public string? Block { get; set; }
+        public string? Parent { get; set; }
+        public string? Type { get; set; }
+        public bool? Append { get; set; } = false;
+        public string? Parameter { get; set; }
+    }
+
+    public class Method
+    {
+        public string[]? Parameters { get; set; }
+        public CodeChange[]? CodeChanges { get; set; }
```

Comment:
anyone can modify the config model? would you be fine with "init" for them to be initialized only at creation time?

Reply:
Not sure what you mean by "init". I don't really need to modify the model after creation but left it there for testing.

Reply:
@abpiskunov